app/i18n/it/sub.php

```diff
@@ -229,8 +229,8 @@
 		'title_add' => 'Aggiungi RSS feed',
 		'ttl' => 'Non aggiornare automaticamente piu di',
 		'unicityCriteria' => array(
-			'_' => 'Article unicity criteria',	// TODO
-			'forced' => '<span title="Block the unicity criteria, even when the feed has duplicate articles">forced</span>',	// TODO
+			'_' => 'Criteri di unicità dell’articolo',
+			'forced' => '<span title="Blocca i criteri di unicità, anche quando il feed ha articoli duplicati">forced</span>',	// DIRTY
```

Comment:
```suggestion
			'forced' => '<span title="Blocca i criteri di unicità, anche quando il feed ha articoli duplicati">forced</span>',
```

Reply:
The term `forced` should be translated but I am not sure how to translate it